flink-runtime-web/web-dashboard/web/css/vendor.css

```diff
@@ -7704,6 +7704,7 @@ button.close {
 .modal-header {
   padding: 15px;
   border-bottom: 1px solid #e5e5e5;
+  min-height: 16.42857143px;
```

Comment:
If I'm not mistaken, then the `vendor.css` file is generated and should not be changed. Css style changes should go to `app/styles/`